### DISCUSSION TITLE
Code First

### DISCUSSION THREAD

**Discussion by MartinEgli** [quartznet/quartznet#1032] [Q&A]
I want to use Quartz.net with Entity Framework / Entity Framework Core Code First. I want to use the database as JobStore. What is the best way to do it with code first?

**Comment by lahma** [comment]
Here's an old answer https://stackoverflow.com/questions/25524624/how-can-i-make-quartz-net-adojobstore-work-with-entity-framework . Generally as schema only changes between major releases creating the schema using scripts is quite straightforward. Quartz currently has no dependencies to ORM libraries.
